docs/mobile-apps/automated-testing/espresso-xcuitest/espresso-capture.md

```diff
@@ -0,0 +1,39 @@
+---
+id: espresso-capture
+title: Espresso Screenshot Capture
+sidebar_label: Espresso Screenshot Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Espresso Screenshot Capture functionality allows you to take screenshots of errors to support the visual verification of your debugging process. These screenshots also help to identify any errors or bugs within your application.
+
+
+## Capturing Screenshots
+
+To take screenshots during an Espresso test run, follow the steps below. Make sure to handle exceptions in your test code to ensure a proper test run. This feature is supported on Android 4.3 and above.
+
+### To capture screenshots:
+
+1. Add the utility class to your project. See our Java example on [GitHub](https://github.com/saucelabs/my-demo-app-android/blob/2daaab68f6b75dcd78533dda7ac1715eec070f99/app/src/androidTest/java/com/saucelabs/mydemoapp/android/screenshot/SauceLabsCustomScreenshot.java).
+2. Add the following code to your Espresso test scripts. See our demo app example on [GitHub](https://github.com/saucelabs/my-demo-app-android/blob/2daaab68f6b75dcd78533dda7ac1715eec070f99/app/src/androidTest/java/com/saucelabs/mydemoapp/android/view/activities/LoginTest.java).
```

Comment:
Can we show a code example on how to use this a Java and Kotlin test, this is extra clicks for users and also they need to search for how to implement this

Something like

```java
// Import the class at the top of your test file
import com.saucelabs.mydemoapp.android.screenshot.SauceLabsCustomScreenshot;

// To take a screenshot use this method inside of your test 
SauceLabsCustomScreenshot.capture("before_login_input");
```

Reply:
Corrected.